auth-web/src/router.ts

```diff
@@ -99,6 +101,8 @@ export function getRoutes () {
     { path: '/unauthorized', name: 'unauthorized', component: UnauthorizedView, props: true, meta: { requiresAuth: false } },
     { path: '/pendingapproval/:team_name?', name: 'pendingapproval', component: PendingApprovalView, props: true, meta: { requiresAuth: false, requiresProfile: true } },
     { path: '/leaveteam', name: 'leaveteam', component: LeaveTeamLandingView, props: true, meta: { requiresAuth: true } },
+    { path: '/setupaccount', name: 'setupaccount', component: SetupAccountView, props: true, meta: { requiresAuth: true, allowedRoles: [Role.Staff] } },
```

Comment:
Can we rename these routes to be more staff specific? Maybe `/staff-setup-account` and `/staff-setup-account-success`?

Reply:
Agreed, will do that

Reply:
'-' is okay.  We use it some of the child routes, i.e. `/account/:orgId/settings/account-info`